tests/integration/components/did-insert-parent-hook-test.js

```diff
@@ -68,18 +77,19 @@ test('top-level parent and two children-parents', function(assert) {
   assert.ok(parentSpy.calledBefore(childSpy), 'parent was called before child');
 });
 
-test('top-level parent and two children-parents after if', function(assert) {
-  let parentSpy = this.parentSpy = this.spy();
-  let childSpy = this.childSpy = this.spy();
-  let childParentSpy = this.childParentSpy = this.spy();
+test('top-level parent with children-parents after if', function(assert) {
```

Comment:
I actually started out with it in a separate test. I moved it here because it wasn't testing a new thing, just clarifying of how the run order should work for parents that were already set up.

I'm happy to split it back out again if you prefer.